src/Qowaiv.Validation.DataAnnotations/README.md

```diff
@@ -23,6 +23,32 @@ var warn = ValidationMessage.Warning(message, args);
 var error = ValidationMessage.Error(message, args);
 ```
 
+## Required modifier
+The [`required`](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/required) modifier is available since C# 11.
+When a model has a property with `required` modifier, it is considered beining
+decorated with a `[Required]` attribute when the property:
+1. is a reference type
+2. lacks a nullable (`?`) annotation
+3. is not decorated with a `[Required]` attribute.
```

Comment:
"It is considered being decorated with a [Required] attribute when the property: is not decorated with a [Required] attribute"

This sentence is confusing, the example makes it more clear, but maybe you can rephrase.

Reply:
Any suggestions on how? I was not sure.